packages/elements/src/components/table/index.tsx

```diff
@@ -25,13 +27,14 @@ const getHeadersFromRows = (rows: Row[]): string[] => {
 export type Cell = {
   label: string
   value: string
-  children: React.ReactNode
+  children?: React.ReactNode
   icon?: IconNames
   statusCircleIntent?: Intent
   cellHasDarkText?: boolean
   narrowTable?: {
```

Comment:
Is it worth defining `narrowTable` as it's own type?

Reply:
can do. Do you know if that argsTable thing works with Interfaces/Types?